AppPrawject/AppPrawject/Views/Pet/Form.cshtml

```diff
@@ -25,7 +50,7 @@
         <input class="form-control" asp-for="Image">
     </div>
 
-    <button type="submit" class="btn btn-primary">Add</button>
+    <button type="submit" class="btn btn-primary">action</button>
```

Comment:
change `<button type="submit" class="btn btn-primary">action</button>` to `<button type="submit" class="btn btn-primary">@action</button>`